src/cpp/session/SessionConsoleInput.cpp

```diff
@@ -101,20 +101,21 @@ void consolePrompt(const std::string& prompt, bool addToHistory)
 
 bool canSuspend(const std::string& prompt)
 {
-   bool suspendIsBlocked = false;
```

Comment:
I thought we intentionally used `|=` below because we do want to know (for each of these operations) whether it's blocking suspend or not, even if there are multiple operations blocking suspend. @cm421 does that sound true?

Reply:
Oh, that makes sense. We are using those methods to track why it won't suspend?  I should have dug into that code in more depth.  I will revert this optimization as it's probably not helpful anyway since most of the time, they all return false.